sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -736,13 +736,15 @@ class SparkSession private(
   }
   // scalastyle:on
 
+  def stop(): Unit = stop(0)
+
   /**
    * Stop the underlying `SparkContext`.
    *
    * @since 2.0.0
```

Comment:
You should have the new docs for the one for `exitCode` with new `@since`

Reply:
How about current?